src/main/java/seedu/address/commons/core/LogsCenter.java

```diff
@@ -60,14 +61,24 @@ public static <T> Logger getLogger(Class<T> clazz) {
     }
 
     /**
-     * Adds the {@code consoleHandler} to the {@code logger}. <br>
-     * Creates the {@code consoleHandler} if it is null.
+     * Creates and returns a base logger with the default configuration with the name {@code BASE_LOGGER_NAME}.
```

Comment:
Also mention the handlers (as they are referenced in the last sentence). Something like this?
```suggestion
     * Creates and returns a base logger with the default configuration with the name {@code BASE_LOGGER_NAME}
     * and has a (@code ConsoleHandler} and a {@code FileHandler}.
```